### ISSUE TITLE
Cannot open pass tomb, "Error: open is not in the password store."

### ISSUE THREAD

**Issue by yankee14** [roddhjav/pass-tomb#38]
I am running Debian Testing. I installed package updates, rebooted my computer, and pass tomb seems to no longer work. It will not let me open my password tomb, and fails with the error message:

```
$ pass open
Error: open is not in the password store.
```

It looks like the `pass` program might be intercepting `pass-tomb`? Or it thinks the store is already open? Also, I am pretty sure I forgot to close the password store before I rebooted, but this has never been a problem before.

```
$ uname -a
Linux [hostname] 5.15.0-2-amd64 #1 SMP Debian 5.15.5-2 (2021-12-18) x86_64 GNU/Linux
```

```
$ apt policy pass pass-extension-tomb
pass:
  Installed: 1.7.4-4
  Candidate: 1.7.4-4
  Version table:
 *** 1.7.4-4 500
        500 http://deb.debian.org/debian testing/main amd64 Packages
        500 http://deb.debian.org/debian testing/main i386 Packages
        100 /var/lib/dpkg/status
pass-extension-tomb:
  Installed: 1.3-1
  Candidate: 1.3-1
  Version table:
 *** 1.3-1 500
        500 http://deb.debian.org/debian testing/main amd64 Packages
        500 http://deb.debian.org/debian testing/main i386 Packages
        100 /var/lib/dpkg/status
```

**Comment by roddhjav** [comment]
Hi, this is a know bug in the Debian packaging: See https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=1003996 It seems it has been fixed in `pass-extension-tomb 1.3-2`.

**Comment by yankee14** [comment]
@roddhjav ah okay, thank you for catching that. Upgrading to 1.3-2 from `unstable` fixed the issue.
